server/index.js

```diff
@@ -1,17 +1,39 @@
 const express = require('express');
 const app = express();
 const path = require('path');
-const fileUpload = require('express-fileupload')
+const fileUpload = require('express-fileupload');
 const { db, models } = require('./db/index.js');
 const { seedUsers, seedProducts } = require('../seed.js');
 const cookieParser = require('cookie-parser');
+const { Session } = models;
 
 app.use(express.json());
-app.use(cookieParser())
+app.use(cookieParser());
 
-app.use(fileUpload({
-  limits: { fileSize: 50 * 1024 * 1024 }
-}))
+app.use((req, res, next) => {
+	if (!req.cookies['session_id']) {
+		Session.create()
+			.then(session => {
+				res.cookie('session_id', session.id,{
+          path: '/',
+          expires: new Date(Date.now() + 1000 * 60 * 60 * 24)
+        });
+				next();
+			})
+			.catch(e => {
+				console.error(e);
+				res.satus(404).redirect('/error');
+			});
+	}else{
+    next()
```

Comment:
Are we going to set some kind of environment variable that lets us know what the session status is for the rest of the app?